### DISCUSSION TITLE
Pass array of string instead of comma separated string to yup resolver

### DISCUSSION THREAD

**Discussion by neilnahid** [react-hook-form/react-hook-form#3464] [Q&A]
### useForm declaration
![image](https://user-images.githubusercontent.com/25801890/99361895-c147cb00-28ed-11eb-8b87-8ee46ac6b1f2.png)

### Input field
![image](https://user-images.githubusercontent.com/25801890/99361773-96f60d80-28ed-11eb-8282-35898ece70b0.png)
yup schema

### Yup Schema
![image](https://user-images.githubusercontent.com/25801890/99361816-a5dcc000-28ed-11eb-8bc6-d0d3e93d4aef.png)

### error when submitting

```
{"skills":{"message":"skills must be a `array` type, but the final value was: `null` (cast from the value `\"test,another\"`).\n If \"null\" is intended as an empty value be sure to mark the schema as `.nullable()`","type":"typeError"}}
```

**Comment by bluebill1049** [comment]
codesandbox plz

**Reply by neilnahid** [reply]
https://codesandbox.io/s/charming-water-ggiss?file=/src/App.tsx

**Reply by bluebill1049** [reply]
i think you need to wrap with Controller: 

https://github.com/react-hook-form/react-hook-form/tree/master/examples
search: Parse and format input value
